dbicdh/PostgreSQL/deploy/89/001-auto-__VERSION.sql

```diff
@@ -0,0 +1,18 @@
+-- 
```

Comment:
It looks wrong that the database migration is still in place while this is now supposed to use cookies.

Reply:
I am not sure about that. Cookies is a user preference. The DB migration is about the preference for a product. Should we removed that? @okurz what do you think?

Reply:
I personally had liked to keep it simple. But if you and the others like to keep both ways I'm ok with it.